src/main/java/com/backbase/billpay/fiserv/ebills/EbillsServiceImpl.java

```diff
@@ -58,9 +60,11 @@ public void disableEbills(Header header, String payeeId) {
     }
     
     @Override
-    public BillPayEbillsGetResponseBody getEbills(Header header, String payeeId, String status, Date startDate,
-                    Date endDate, Integer from, Integer size, String orderBy, String direction) {
-        
+    public BillPayEbillsGetResponseBody getEbills(Header header, String payeeId, String status, LocalDate sDate,
+                    LocalDate eDate, Integer from, Integer size, String orderBy, String direction) {
+        Date startDate = fromLocalDate(sDate);
```

Comment:
There's no need to convert these into Dates as when used they are only converted into FiservDates which are date-only already. You can convert from LocalDate to FiservDate directly.